src/components/Button/Button.tsx

```diff
@@ -44,21 +52,28 @@ const Button: React.FC<ButtonProps> = ({
   outlineWidth,
   borderRadius,
   disabled,
+  labelProps,
   ...props
-}) => (
-  <ButtonWrapper
-    onPress={onPress}
-    activeOpacity={0.8}
-    backgroundColor={backgroundColor}
-    outline={outline}
-    outlineColor={outlineColor}
-    outlineWidth={outlineWidth}
-    borderRadius={borderRadius}
-    disabled={disabled}
-    {...props}>
-    <Label color={textColor}>{text}</Label>
-  </ButtonWrapper>
-)
+}) => {
+  const ButtonTheme = useTheme().components.Button
+  return (
+    <ButtonWrapper
+      onPress={onPress}
+      activeOpacity={0.8}
```

Comment:
should we need to include value activeOpacity in theme as well?

Reply:
We actually don't need to, the props is extends from `TouchableOpacityProperties` so it already have `activeOpacity`. And since we pass all the props into with `{...props}`, if `activeOpacity` exist, it will override the above value